docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the currently assigned Freight. |
```

Comment:
"assigned Freight" sounds awkward. Freight gets _promoted_ to a Stage. We don't really speak in terms of Freight being _assigned_ to a Stage. I think the easiest resolution here would be this:

```suggestion
| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the current Freight. |
```